packages/app/cypress/e2e/specs_list_actual_git_repo.cy.ts

```diff
@@ -2,10 +2,10 @@ describe('Spec List - Git Status', () => {
   beforeEach(() => {
     cy.scaffoldProject('cypress-in-cypress')
     .then((projectPath) => {
-      cy.task('initGitRepoForTestProject', projectPath)
       cy.wait(500)
```

Comment:
If we're moving stuff around, is this wait still needed?

Reply:
Nope! [4bd8b90](https://github.com/cypress-io/cypress/pull/21212/commits/4bd8b90c2bb90d6c2e3dd6b11a74e6f89edcc386)